modules/nw-create-custom-ingress-controller.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-create-custom-ingress-controller_{context}"]
+= Creating a custom Ingress Controller
+
+As a cluster administrator, you can create a new custom Ingress Controller. Because the default Ingress Controller might be updated during {product-title} upgrades, creating a custom Ingress Controller can be helpful when maintaining a configuration manually that persists across cluster upgrades.
```

Comment:
We [want](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update) update rather than upgrade
```suggestion
As a cluster administrator, you can create a new custom Ingress Controller. Because the default Ingress Controller might change during {product-title} updates, creating a custom Ingress Controller can be helpful when maintaining a configuration manually that persists across cluster updates.
```